### ISSUE TITLE
Installation mystery - iATS does not appear as a Payment Processor choice

### ISSUE THREAD

**Issue by joemcl** [iATSPayments/com.iatspayments.civicrm#66]
So I've added the iATS 1.3.1 module to my CiviCRM modules folder. Eileen installed the needed patch. We are running SOAP. But, I don't see iATS as an option under Administer/System Settings/Payment Processors. Did I miss something on the install?

**Comment by KarinG** [comment]
Did you install the extension on the Manage Extensions screen?

**Comment by joemcl** [comment]
Karin, no, because I've installed modules/extensions direct to the server via SFTP successfully before, and in any event, the version listed in Civi at CiviCRM Extensions is 1.2.11, not the latest and recommended 1.3.1, which I downloaded from https://civicrm.org/extensions/iats-payments/version-131. I don't want to install an older version.

**Comment by KarinG** [comment]
It's ok to get your source code that way -> but you still need to go to the Manage Extensions screen to install the extension: /civicrm/admin/extensions?reset=1 -> it needs to have status = Enabled. 
(this will create the entities, the tables in the database etc).

**Comment by joemcl** [comment]
OK. Will try that. Could somebody update the info being fed from http://civicrm.org/extensions to the CiviCRM Extensions page if this is correct - "[t]his page will automatically list the available 'native' extensions from the CiviCRM.org extensions directory which are compatible with this version of CiviCRM" is correct.

**Comment by KarinG** [comment]
We have 1.3.1 uploaded/listed on https://civicrm.org/extensions/iats-payments

Fully Qualified Name: 
com.iatspayments.civicrm
Current Usage: 
122
Releases:
Version 	Date 	Status 	CiviCRM Compatibility
Version 1.3.1 	Dec 19 2014 	Stable 	CiviCRM 4.4 - 4.5
Version 1.2.11 	Sep 11 2014 	Stable 	CiviCRM 4.2 - 4.4
Version 1.2.10 	Sep 4 2014 	Stable 	CiviCRM 4.2 - 4.4
Version 1.2.9 	Aug 17 2014 	Stable 	CiviCRM 4.2 - 4.4
Version 1.2.8 	Aug 5 2014 	Stable 	CiviCRM 4.2 - 4.4 

So I'm not sure why your CiviCRM is not seeing it. In any case, you always must install an extension regardless of where you get the source code from.

http://wiki.civicrm.org/confluence/display/CRMDOC/Extensions:
See: Manual installation of native extensions

**Comment by KarinG** [comment]
You're not on CiviCRM 4.2 or 4.3 are you?

**Comment by joemcl** [comment]
Karin, thanks.So far so good. We are on Civi 4.4.13.

**Comment by adixon** [comment]
If you're on 4.4.x and 1.3.1 is not showing up, that's weird. I know there have been some recent updates to the tagging system for extensions, i wonder if that's getting in the way. You did refresh? 1.3.1 has been available for and in use with 4.4 (and 4.5) for several months now. If your extensions listing only shows an earlier version of the iats, then there's something funky with your setup I expect.

**Comment by joemcl** [comment]
Alan, thanks, checking out why refresh did not show the latest version. 

Meanwhile - this is the latest challenge.  I installed the iATS extension successfully, with a little help from Karin. Then added iATS as a Payment Processor.

I set up a test CiviCRM contributions page - called iATS EFT Test Page, using the profile Name and Address.

I was able to open the page once in incognito mode and attempt a contribution, but got an error. 

I removed and then added back in County from the profile Name and Address to see if that was the cause (didn't work anyway to select County). 

But the test drive and live pages crash now. 

Error message is "Fatal error: require_once(): Cannot redeclare class insertquery_mysql in /var/www/citizenactionny/my.citizenactionny.org/includes/database/database.inc on line 1777"

Any thoughts on a fix for the crash? I'm looking at the offending line 1777 for some clues. I found this post on Drupal with a similar error message - https://www.drupal.org/node/838744 . Checking that out.
